.github/workflows/update-semgrep-rules.yml

```diff
@@ -0,0 +1,59 @@
+name: Update Semgrep Rules
+on:
+  workflow_dispatch: {}
+  # TODO: Remove this trigger
```

Comment:
actually I like to also have it, so it's easy to test improvement to the cron by pushing to this special cron branch (we do that for our parsing-stat and naming-stat crons and it's convenient instead of waiting for the cron to kick in at the scheduled time)

Reply:
why on pull_request was not working?